lib/replaceRef.js

```diff
@@ -14,8 +14,20 @@ module.exports = replaceRef;
 * @param  {string} basePath - The root path where any relative schema file references can be resolved
 * @return {object} The schema object with all schema file referenced replaced with the actual file content.
 */
+var idmap = {}
```

Comment:
Putting this here breaks the jshint documentation above it (it's no longer associated with the function definition).